apps/prepopulate/data_init/vocabularies.json

```diff
@@ -6,13 +6,28 @@
         "selection_type": "multi selection",
         "unique_field": "qcode",
         "items": [
+
             {"is_active": true, "name": "FIXME1", "qcode": "f1", "subject": "", "translations": {"name": {"fr": "FIXME1-fr", "es": "FIXME1-es"}}},
             {"is_active": true, "name": "FIXME2", "qcode": "f2", "subject": "", "translations": {"name": {"fr": "FIXME2-fr", "es": "FIXME2-es"}}}
         ],
         "schema": {
             "name": {"translate": true},
             "qcode": {},
             "subject": {}
+        },
+        "translations": {
+            "display_name": {
+                "ar": "فئات",
+                "da": "kategorier",
+                "de_DE": "Kategorien",
```

Comment:
would be good to be consistent with those language keys, I think the idea is to stick with `de-DE` like format